Makefile

```
@@ -7,15 +7,25 @@ check:
 lint:
 	pipenv run flake8
 
-test: check lint
-	pipenv run pytest test --cov app --cov-report term-missing
+unit_tests: check lint
+	 pipenv run pytest test/unit_tests --cov app --cov-report term-missing
 
+integration_tests:
+	docker-compose down
+	docker-compose up
```

Should fix travis issue: 

```suggestion
	docker-compose up -d
```

---

Maybe change Makefile to:

```diff
diff --git a/Makefile b/Makefile
index 21168a4..af333bf 100644
--- a/Makefile
+++ b/Makefile
@@ -10,10 +10,7 @@ lint:
 unit_tests: check lint
         pipenv run pytest test/unit_tests --cov app --cov-report term-missing

-integration_tests:
-       docker-compose down
-       docker-compose up
-       bash ./test/integration_tests/wait_for_print_file_service.sh
+integration_tests: down up
        pipenv run pytest test/integration_tests
        docker-compose down

@@ -28,6 +25,7 @@ build_and_test: install unit_tests docker_build integration_tests

 up:
        docker-compose up -d
+       bash ./test/integration_tests/wait_for_print_file_service.sh

 down:
        docker-compose down
```

---

@jcox-dev The print file service can't connect to the SFTP in travis, we temporarily removed the `-d` option to see all the start up logs.